docs/design/input.md

```diff
@@ -7,29 +7,32 @@ Github audit plugin will be using config.yaml file for taking input from user
 ## provides logging level <OPTIONAL> , Default: info
 loglevel: debug
 ## folder path to log file <OPTIONAL> , Default: same as git-audit binary location 
-logpath: ./test/ 
+logpath: ./test.log 
 auditJobs:
 ## audit job name <REQUIRED>
 - name: auditjob1
-## polling interval to fetch data to be defined in cron job format  <REQUIRED> , Default: 5 * * * *   
-  polling_interval: 5 * * * *   
+## polling interval to fetch data to be defined  <REQUIRED> , Default: 5s, 10m etc  
```

Comment:
default should be single value. option can be 5s, 10m etc

Reply:
changing it